### PR TITLE
upd run_validator

### PR DESCRIPTION
**Pull request by mrlp4**
Updated run_validator doc for "out-of-docker" usage.

### CODE REVIEW

File: docs/run_validator.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
 
 # Join Cyberd testnet as a Validator
 
-**Note** The current active testnet is `euler-5` (substitute <testnet_chain_id> with that value, do not forget to remove the `<` and the `>` symbols).
+**Note** The current active testnet is `euler-6` (substitute <testnet_chain_id> with that value, do not forget to remove the `<` and the `>` symbols).
 
 ## Prepare your server
```
Comment (litvintech) [inline docs/run_validator.md]: add note on dedicated server for stability @mrlp4

File: docs/run_validator.md [hunk 2]
```diff
@@ -225,157 +178,235 @@ You should see this:
 +-----------------------------------------------------------------------------+
 ```
 
-#### Install Nvidia container runtime for docker
+### Install CUDA toolkit
+
+Simply run `apt install nvidia-cuda-toolkit`. Any version above 9.1 is OK. To check the version run `nvcc --version`.
```
Comment (litvintech) [inline docs/run_validator.md]: add check on nvcc installation @mrlp4

File: docs/run_validator.md [hunk 3]
```diff
@@ -225,157 +178,235 @@ You should see this:
...
+cyberdcli keys show <your_key_name>
 ```
 
-You could use your ledger device with the Cosmos app installed on it to sign and store cyber addresses: [guide here](https://github.com/cybercongress/cyberd/blob/0.1.5/docs/cyberd_Ledger_guide.md).
+You could use your Ledger device, with the Cosmos app installed on it to sign and store cyber addresses: [guide here](https://github.com/cybercongress/cyberd/blob/0.1.5/docs/cyberd_Ledger_guide.md).
 In common case use the --ledger flag, with your commands:
 
 ```bash
-docker exec -ti euler-5 cyberdcli keys add <your_key_name> --ledger
+cyberdcli keys add <your_key_name> --ledger
```
Comment (litvintech) [inline docs/run_validator.md]: add notes to use with keyring and local file, not only with ledger @mrlp4

File: docs/run_validator.md [hunk 4]
```diff
@@ -225,157 +178,235 @@ You should see this:
...
+
 ```bash
-curl -s -L https://nvidia.github.io/nvidia-docker/gpgkey | sudo apt-key add -
+mkdir $HOME/.cyberd
+mkdir -p $DAEMON_HOME/upgrade_manager
+mkdir -p $DAEMON_HOME/upgrade_manager/genesis
+mkdir -p $DAEMON_HOME/upgrade_manager/genesis/bin
 ```
 
+3. Get a prebuilt copy of cosmosd or build it from source:
```
Comment (litvintech) [inline docs/run_validator.md]: add note on current link if upgrade exist

Comment (litvintech) [inline docs/run_validator.md]: add note on current link if upgrade exist @mrlp4

File: docs/run_validator.md (hunk 1 above)
Comment (mrlp4) [reply docs/run_validator.md]: add note.

File: docs/run_validator.md (hunk 2 above)
Comment (mrlp4) [reply docs/run_validator.md]: checked

File: docs/run_validator.md (hunk 4 above)
Comment (mrlp4) [reply docs/run_validator.md]: done

File: docs/run_validator.md (hunk 3 above)
Comment (mrlp4) [reply docs/run_validator.md]: done

**Pull request merged**